conversant-htb.js

```diff
@@ -90,8 +204,9 @@ function ConversantHtb(configs) {
      */
     function __generateRequestObj(returnParcels) {
         var queryObj = {};
-        var baseUrl = Browser.getProtocol() + '';
-        var callbackId = System.generateUniqueId();
+        
+        /* todo : specify length and format for cache buster */
+        var baseUrl = Browser.getProtocol() + 'media.msg.dotomi.com/s2s/header/24?cb=' + System.generateUniqueId();
```

Comment:
Please update such that there are double slashes

'//media.msg.dotomi.com/s2s/header/24?cb='

Reply:
In addition, please make the following changes to the mock library file for the unit test.
https://github.com/indexexchange/conversant-adapter/blob/master-v2/spec/support/libraryStubData.js#L11-L27
line 12: return 'http://'; = > return 'http:'; (reason why '//' change is needed)
line 18: return 'localhost'; => return 'http://localhost';
line 21: return 'desktop'; => return 'Mozilla/5.0 (Windows; U; Windows NT 6.1; rv:2.2) Gecko/20110201';
Thanks